Source/ScrollViewAdapters/EmptyContentScrollViewAdapter.swift

```diff
@@ -166,4 +208,50 @@ open class EmptyContentScrollViewAdapter<State: Equatable, View: UIScrollView>:
         self.button()?.action()
     }
     
+    /**
+    
+     Delegate function of DZNEmptyDataSet if empty content should allow touch
+     
+     - parameter _: the scrollView subclass to display the empty content.
+     
+    */
+    open func emptyDataSetShouldAllowTouch(_ scrollView: UIScrollView!) -> Bool {
+        guard let shouldAllowTouch = emptyContentPresentation?.shouldAllowTouch(for: stateManaging.state) else {
```

Comment:
could be shortened  by using `return emptyContentPresentation?.shouldAllowTouch(for: stateManaging.state) ?? false`

Not sure if it is better